geo_project_compassion/models/project_compassion_geoengine.py

```diff
@@ -0,0 +1,32 @@
+from odoo import api
+from odoo.addons.base_geoengine import geo_model
+from odoo.addons.base_geoengine.fields import GeoPoint
+
+
+class CompassionProject(geo_model.GeoModel):
+    """Add geo_point to compassion.project"""
+
+    _inherit = "compassion.project"
+
+    geo_point = GeoPoint(
+        readonly=True, store=True)
```

Comment:
I believe store is not necessary unless they defined GeoPoint as not stored in database by default?